eng/jobs/linux-build.yml

```diff
@@ -0,0 +1,178 @@
+parameters:
+  Jobs: []
+
+jobs:
+- ${{ each job in parameters.Jobs }}: 
+  - job: ${{ job.displayName }}
+    displayName: ${{ job.displayName }}
+    pool: ${{ job.pool }}
+    strategy: ${{ job.strategy }}
+    variables: 
+      ${{ insert }}: ${{ job.variables }}
+
+      CommonMSBuildArgs: /flp:v=diag /clp:v=detailed 
+        /p:TargetArchitecture=x64
+        /p:PortableBuild=false
+        /p:ConfigurationGroup=$(_BuildConfiguration)
+        /p:OSGroup=Linux
+        /p:OfficialBuildId=$(OfficialBuildId)
+        /p:StabilizePackageVersion=$(IsStable)
+
+      SourcesDirectory: '$(Build.SourcesDirectory)'
+
+      # Packaging related variables
+      # Debian
+      CommonDebianRunCommand: run --rm --name centos-7-d485f41-20173404063424ubuntu-14.04-debpkg-e5cf912-20175003025046 
```

Comment:
Can we not use the native azure devops container support here?  That avoids a whole lot of complexity in managing containers.

Reply:
@chcosta  did share the container support details. Per https://github.com/Microsoft/azure-pipelines-yaml/blob/master/design/container-steps.md , this feature is on hold . Also, implementing this latest design change would impact the delivery date. It can be taken care of in next iteration of work pertaining to Arcade.